src/lib/backend/services/authService.ts

```
@@ -0,0 +1,97 @@
+    }
+  }
+
+  // 회원 탈퇴
+  static async withdraw(): Promise<void> {
+    const { error } = await client.DELETE("/api/members/withdraw");
+
+    if (error) {
+      throw new Error("회원 탈퇴에 실패했습니다.");
+    }
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)